sky/specs/style-guide.md

```diff
@@ -1,5 +1,4 @@
-Flutter Style Guide
-===================
+# Flutter Style Guide
```

Comment:
Consistency contributes to readability big time. But we're talking about a couple headlines here, so I'll revert the change.